client-py/README.md

```diff
@@ -353,6 +353,60 @@ class MyTestCase(unittest.TestCase):
 by default it will load the image `apache/iotdb:latest`, if you want a specific version just pass it like e.g. `IoTDBContainer("apache/iotdb:0.12.0")` to get version `0.12.0` running.
 
 
+### IoTDB DBAPI
+
+IoTDB DBAPI implements the Python DB API 2.0 specification (https://peps.python.org/pep-0249/), which defines a common
+interface for accessing databases in Python.
```

Comment:
And consider adding this to https://github.com/apache/iotdb/blob/master/docs/zh/UserGuide/API/Programming-Python-Native-API.md and https://github.com/apache/iotdb/blob/master/docs/UserGuide/API/Programming-Python-Native-API.md

Reply:
added.